src/library/DIPS.Mobile.UI/Components/FloatingActionButtons/FloatingNavigationButton/NavigationMenuButton/NavigationMenuButton.Properties.cs

```diff
@@ -0,0 +1,110 @@
+using System.Windows.Input;
+using Colors = DIPS.Mobile.UI.Resources.Colors.Colors;
+
+namespace DIPS.Mobile.UI.Components.FloatingActionButtons.FloatingNavigationButton.NavigationMenuButton;
+
+internal partial class NavigationMenuButton
+{
+    /// <summary>
+    /// Sets the icon
+    /// </summary>
+    public ImageSource Icon
+    {
+        get => (ImageSource)GetValue(IconProperty);
+        set => SetValue(IconProperty, value);
+    }
+
+    /// <summary>
+    /// Set the button's background color
+    /// </summary>
+    public Color ButtonBackgroundColor
+    {
+        get => (Color)GetValue(ButtonBackgroundColorProperty);
+        set => SetValue(ButtonBackgroundColorProperty, value);
+    }
+
+    /// <summary>
+    /// Set the icon's rotation
+    /// </summary>
+    public float IconRotation
+    {
+        get => (float)GetValue(IconRotationProperty);
+        set => SetValue(IconRotationProperty, value);
+    }
+
+    /// <summary>
+    /// Set the command
+    /// </summary>
+    public ICommand Command
+    {
+        get => (ICommand)GetValue(CommandProperty);
+        set => SetValue(CommandProperty, value);
+    }
+
+    /// <summary>
+    /// Set the badge count
+    /// <remarks>If the badge count is zero, the badge will not be visible</remarks>
+    /// </summary>
+    public int BadgeCount
+    {
+        get => (int)GetValue(BadgeCountProperty);
+        set => SetValue(BadgeCountProperty, value);
+    }
+
+    /// <summary>
+    /// Set the color of the badge
+    /// </summary>
+    public Color BadgeColor
+    {
+        get => (Color)GetValue(BadgeColorProperty);
+        set => SetValue(BadgeColorProperty, value);
+    }
+
+    /// <summary>
+    /// Set its state to disabled, will make the opacity .5 and not tappable
+    /// </summary>
+    public bool Disabled
```

Comment:
Samme som forrige kommentar ang IsEnabled

Reply:
Blir det ikke vanskelig å vite hva IsEnabled gjør for den komponenten?